TotalSegmentator/TotalSegmentator.py

```diff
@@ -621,7 +640,11 @@ def setupPythonRequirements(self, upgrade=False):
         if needToInstallSegmenter or upgrade:
             self.log('TotalSegmentator Python package is required. Installing... (it may take several minutes)')
 
-            if upgrade:
+            if self.pullMaster: 
+                slicer.util.pip_uninstall("TotalSegmentator")
```

Comment:
This may fail if TotalSegmentator is already used in this session. If it indeed fails it would be good to explain to the user that this is normal and the user should restart Slicer and force reinstallation again to complete the operation.

Reply:
I do not understand this. Where would something fail ? All my forced reinstallations never failed. 

Are you suggesting to catch an exception ? Please suggest a solution.